packages/nimble-components/build/generate-workers/tests/render-worker.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { HealthStatus } from '../source/health-status';
+import { RenderWorker } from '../source/render-worker';
+
+describe('RenderWorker', () => {
+    it('worker should be healthy', () => {
+        const renderer = new RenderWorker();
+        expect(renderer.isWorkerHealthy()).toBe(HealthStatus.Healty);
```

Comment:
can you check if the worker is healthy after wrapping it with comlink like you did in the renderer?